src/main/kotlin/com/atlassian/performance/tools/virtualusers/LoadSegment.kt

```diff
@@ -16,6 +16,5 @@ internal class LoadSegment(
     override fun close() {
         done.set(true)
         output.close()
-        driver.close()
```

Comment:
Controversial, because we sacrifice the cleanliness of local VU runs. Does anyone have a better idea?

Reply:
IMO the overhead doesn't have to be outside of our control. We can stop it from a thread and wait for a close on a Future.

Reply:
Async/sync doesn't solve the problem, because it still needs to be done while the JVM is alive. And we count the JVM duration via `ssh.execute("java -jar ...")`

Reply:
And if we do it in a daemon thread, then it means the JVM can die before it's done and therefore it won't be done. So **if** we're fine with browsers not being cleaned up *sometimes*, perhaps we should fine skipping the cleanup *always*.

Reply:
It doesn't solve the problem, but we can control how long to wait for the close. It means we can fail when we fail to close in time, instead of skipping close every time.

Reply:
Ok, so we can add a best-effort cleanup and control the pessimistic case. I like it.

Reply:
I'll use 30 seconds as the timeout. Apart from very spiky outliers, our regression test results max out at around 18 second action times: 
<img width="1248" alt="image" src="https://user-images.githubusercontent.com/1194704/56668378-bac08580-66af-11e9-8b37-add727ee27a7.png">

If the VU is unlucky enough to be performing the 35 second action (1 out of 4509 in this case) while we're shutting down the VU *and* being in the first 5 seconds of this action at that point, so be it.

Reply:
We can also paramterize this timeout.
Pros:
* different timeouts for different use cases (kind of an opt-out too)
* ability to contribute to the `SshVirtualUsers` `ssh.execute` timeout

Cons:
* new API

Worth it or not just yet?

Reply:
Pushed in WD cleanup, with a 30 second timeout, in parallel